<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>399</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Re_Sync</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>210</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Мин. длительность фразы</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spinMinDuration">
            <property name="toolTip">
             <string>Минимальная длительность фразы. Более короткие фразы при группировке игнорируются.</string>
            </property>
            <property name="suffix">
             <string>мс</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Макс. отступ внутри группы</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinMaxOffset">
            <property name="toolTip">
             <string>Максимально допустимый отступ между фразами, при достижении которого создаётся новая группа</string>
            </property>
            <property name="suffix">
             <string>мс</string>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="value">
             <number>5000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Макс. расхождение фраз</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinMaxDesync">
            <property name="toolTip">
             <string>Максимально допустимое расхождение фраз, при котором они ещё считаются синхронными</string>
            </property>
            <property name="suffix">
             <string>мс</string>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Макс. допустимый сдвиг</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinMaxShift">
            <property name="toolTip">
             <string>Максимально допустимое время, на которое может быть сдвинута группа фраз при синхронизации</string>
            </property>
            <property name="suffix">
             <string>мс</string>
            </property>
            <property name="maximum">
             <number>600000</number>
            </property>
            <property name="value">
             <number>15000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="chbSkipComments">
          <property name="maximumSize">
           <size>
            <width>222</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Фильтровать комментарии</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbSkipLyrics">
          <property name="maximumSize">
           <size>
            <width>222</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Фильтровать лирику</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="btOpenSynced">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Открыть синхронный скрипт</string>
          </property>
          <property name="text">
           <string>Открыть синхронный скрипт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btOpenDesynced">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Открыть рассинхронизированный скрипт</string>
          </property>
          <property name="text">
           <string>Открыть рассинхронизированный скрипт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSvg">
          <property name="text">
           <string>Сохранить график</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btAutosync">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Синхронизировать автоматически</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
